scripts/deploy.sh

```diff
@@ -56,12 +56,14 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # move OSA conf files to correct locations
+    mv /etc/openstack_deploy/user_variables.yml /etc/openstack_deploy/user_osa_variables_defaults.yml
```

Comment:
we have to rename it because of the way we are doing precedence (by name).

user_variables overrides user_rpc_variables.